lib/hanami/cli.rb

```diff
@@ -1,37 +1,66 @@
-require "hanami/cli/version"
-require "hanami/cli/renderer"
-require "hanami/cli/command/params_parser"
-
 module Hanami
   class Cli
-    attr_reader :commands
-    attr_reader :renderer
+    require "hanami/cli/version"
+    require "hanami/cli/command"
+    require "hanami/cli/registry"
+    require "hanami/cli/parser"
+    require "hanami/cli/usage"
+    require "hanami/cli/banner"
```

Comment:
Because if you do at the top of the file, then the first definition of `Hanami::Cli` it isn't by `hanami/cli.rb`, but by something else.